### PR TITLE
Add clarifying explanation for computed properties

### PR DESCRIPTION
**Pull request by runningcode**
When using a computed property with a `get()` the behavior will be different than without using `get()` so add some explanation for this.

Feel free to help me out with the wording and syntax. :)

### CODE REVIEW

File: pages/docs/reference/properties.md [hunk 1]
```diff
@@ -115,6 +115,28 @@ var setterWithAnnotation: Any? = null
...
+val isEmpty get() = this.size == 0 // Computed on every access.
+val isEmpty = this.size == 0 // Computed once and stored.
+```
+</div>
+
+When using a `var` the behavior is almost the same except that the computed value is stored until it changes or is overwritten.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+``` kotlin
+var isEmpty get() = this.size == 0  // Computed on every access.
```
Comment (jonnyzzz) [inline pages/docs/reference/properties.md]: Setter or an initializer is missing here. It will not compile.

**Comment by runningcode** [comment]
@jonnyzzz Thanks for pointing that out. How does it look now?

File: pages/docs/reference/properties.md [hunk 2]
```diff
@@ -115,6 +115,29 @@ var setterWithAnnotation: Any? = null
 ```
 </div>
 
+
+### Computed Properties
+
+When using a computed property, the value is computed once and stored. This behavior changes when using adding `get()`. The `get()` causes the value to be computed on every access.
```
Comment (jonnyzzz) [inline pages/docs/reference/properties.md]: when using addin`. It seems one word is an extra here

File: pages/docs/reference/properties.md [hunk 3]
```diff
@@ -115,6 +115,29 @@ var setterWithAnnotation: Any? = null
...
+val isEmpty get() = this.size == 0 // Computed on every access.
+val isEmpty = this.size == 0 // Computed once and stored.
+```
+</div>
+
+When using a `var` the behavior is almost the same except that the computed value is stored until it changes or is overwritten.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+``` kotlin
+var isEmpty = false
```
Comment (jonnyzzz) [inline pages/docs/reference/properties.md]: could you please avoid using backing fields here, which are discussed below. It looks like you may have a setter declared too

File: pages/docs/reference/properties.md (hunk 2 above)
Comment (jonnyzzz) [reply pages/docs/reference/properties.md]: What is the computed property? Is it the one with `get` and `set` declared?

**Comment by yole** [comment]
Thanks for your contribution! Unfortunately the text in this PR does not exactly match the terminology we're using (a property with an initializer and no getter isn't a computed property), so I've pushed a different change that makes the role of custom accessors clearer.

**Pull request closed without merging**